src/promptflow/tests/executor/unittests/processpool/test_line_execution_process_pool.py

```diff
@@ -154,27 +153,55 @@ def execute_in_spawn_mode_subprocess(
                         )
 
 
-@pytest.mark.unittest
-class TestLineExecutionProcessPool:
-    def create_line_execution_process_pool(self, dev_connections):
-        executor = FlowExecutor.create(
-            get_yaml_file(SAMPLE_FLOW),
-            dev_connections,
-            line_timeout_sec=1,
-        )
-        run_id = str(uuid.uuid4())
-        bulk_inputs = get_bulk_inputs()
-        nlines = len(bulk_inputs)
-        line_execution_process_pool = LineExecutionProcessPool(
-            executor,
-            nlines,
-            run_id,
-            "",
-            False,
-            None,
+def create_line_execution_process_pool(dev_connections):
+    executor = FlowExecutor.create(
+        get_yaml_file(SAMPLE_FLOW),
```

Comment:
For our timeout specific tests, we'd better not use the sample flow , so we can
1 run the flow without any connections;
2 More control on the timeout seconds. (e.g. sleep 60 seconds in the flow, and wait for 10 seconds)

Maybe update this in another PR

Reply:
Ok